src/System.Private.ServiceModel/tests/Scenarios/Client/ClientBase/ClientBaseTests.cs

```diff
@@ -125,31 +125,92 @@ public static void ClientMessageInspector_Verify_Invoke()
 
     [Fact]
     [OuterLoop]
-    public static void ClientBaseOfT_Call()
+    public static void ClientBaseOfT_Sync_RoundTrip_Check_CommunicationState()
```

Comment:
I actually got rid of most of them; if you see any scenario tests other than this one with CommunicationState checks, it was probably a bad push. I agree that for most of the rest of the tests, they are overkill and hence are gone. 

In this case, I will be a little rigid and keep this one. We do not have a test that tests the CommunicationObject, and CommunicationObject itself can't be tested unless we want to mock up our own derived CommunicationObject and test that (it might not be a bad idea, I'm just not seeing it as a high pri right now) 

I'm being a little lazy in this case, and having a round-trippable ICommunicationObject (in this case, ClientBase) where you can actually get to the `Opened` state is valuable.